ush/config.sh.3DRTMA_dev1

```diff
@@ -75,21 +75,29 @@ SFCOBS_USELIST="/home/amb-verif/ruc_madis_surface/mesonet_uselists"
 #
 QUILTING="TRUE"
 CCPP_PHYS_SUITE="FV3_HRRR"
+
+EXTRN_MDL_ICS_OFFSET_HRS="0"
+BOUNDARY_LEN_HRS="6"
+BOUNDARY_LONG_LEN_HRS="6"
+LBC_SPEC_INTVL_HRS="1"
+EXTRN_MDL_LBCS_OFFSET_HRS="0"
+EXTRN_MDL_LBCS_SEARCH_OFFSET_HRS="0"
+
+#INITIAL_CYCLEDEF="00 00,12 * * 2021 *"
+#BOUNDARY_CYCLEDEF="00 00-02/01,04-08/01,10-14/01,16-20/01,22,23 * * 2021 *"
+#BOUNDARY_LONG_CYCLEDEF="00 03,09,15,21 * * 2021 *"
+#PREP_COLDSTART_CYCLEDEF="00 00,12 * * 2021 *"
+PREP_WARMSTART_CYCLEDEF="00 00-23/01 * * 2021 *"  ##very important, no cold start in RTMA
+ANALYSIS_CYCLEDEF="00 00-23/01 * * 2021 *"
+FORECAST_CYCLEDEF="00 00-23/01 * * 2021 *"
+
 FCST_LEN_HRS="0" #the number of run_post jobs #lbcs
 FCST_LEN_HRS_CYCLES=(1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1)
-LBC_SPEC_INTVL_HRS="1"
-EXTRN_MDL_LBCS_OFFSET_HRS="1"
 DA_CYCLE_INTERV="1" #need to be 1 to have hourly DA
-RESTART_INTERVAL="1" #"1 2 3 4 5 6" #restart file used by GSI, don't need this, how to turn off writing restart files?
+RESTART_INTERVAL="1" #"1 2 3 4 5 6"
```

Comment:
Please remove the comment here. It's unclear and confusing for RTMA purposes.

Reply:
Done